libs/app-updater/main/src/lib/app-updater.service.ts

```diff
@@ -9,11 +9,16 @@ import logger from "electron-log/main"
 import { isAxiosError } from "axios"
 import semver from "semver/preload"
 import { AppHttpService } from "app-utils/main"
-import { AppResult, AppResultFactory, mapToAppError } from "app-utils/models"
+import {
+  AppError,
+  AppResult,
+  AppResultFactory,
+  mapToAppError,
+} from "app-utils/models"
 import { AppUpdaterState } from "app-updater/models"
 
 enum AppProgressBarState {
-  Indeterminate = 2,
+  Indeterminate = 0,
```

Comment:
Regarding the [documentation](https://www.electronjs.org/docs/latest/tutorial/progress-bar), this is not a proper value for indeterminate state

> Setting the parameter to negative values (e.g. -1) will remove the progress bar. Setting it to a value greater than 1 will indicate an indeterminate progress bar in Windows or clamp to 100% in other operating systems.

However, I see it works only for windows, so I'd change it to something like

```suggestion
  Indeterminate = os.platform() === "win32" ? 2 : 0,
```